src/contrib/cluster/Akka.DistributedData/ORMultiValueDictionary.cs

```diff
@@ -325,13 +329,13 @@ public IReplicatedData Merge(IReplicatedData other)
             {
                 if (other is ORMultiValueDictionaryDelta d)
                 {
-                    return new ORMultiValueDictionaryDelta((ORDictionary<TKey, ORSet<TValue>>.IDeltaOperation)Underlying.Merge(d.Underlying));
+                    return new ORMultiValueDictionaryDelta((ORDictionary<TKey, ORSet<TValue>>.IDeltaOperation)Underlying.Merge(d.Underlying), WithValueDeltas || d.WithValueDeltas);
                 }
 
-                return new ORMultiValueDictionaryDelta((ORDictionary<TKey, ORSet<TValue>>.IDeltaOperation)Underlying.Merge(other));
+                return new ORMultiValueDictionaryDelta((ORDictionary<TKey, ORSet<TValue>>.IDeltaOperation)Underlying.Merge(other), WithValueDeltas);
             }
 
-            public IDeltaReplicatedData Zero => ORMultiValueDictionary<TKey, TValue>.Empty;
+            public IDeltaReplicatedData Zero => WithValueDeltas ? ORMultiValueDictionary<TKey, TValue>.EmptyWithValueDeltas : ORMultiValueDictionary<TKey, TValue>.Empty;
```

Comment:
Needed this in order to make `DataEnvelope` merging work correctly for `ORMultiValueDictionary` to work correctly inside the `Replicator`